webapp/src/modules/mapBuilder.js

```diff
@@ -173,9 +173,12 @@ export default class MapBuilder {
       .locate({
         watch: true,
         maxZoom: 20,
+        timeout: 20000,
         enableHighAccuracy: true,
       })
-      .on("locationerror", (e) => onGetLocationError(e))
+      .on("locationerror", (e) => {
+        if (e.code === (1 || 2)) return onGetLocationError();
```

Comment:
`error code 3` is for `timeout` but location service keeps attempting and seems to always prevail so there's no need for throwing geolocation error msg imo

Reply:
Ah, ok. Let's add this to the module:

```js
/** 
 * Error code 3 is for timeout but location service keeps attempting
 * and seems to always prevail so there's no need for throwing geolocation error msg.
 */
function ignoreLocationError(e) {
  return e.code !== 1 && e.code !== 2;
}
```

And use that in your handler.